x11-packages/godot/build.sh

```diff
@@ -34,6 +34,7 @@ termux_step_make() {
 		alsa=no \
 		execinfo=yes \
 		pulseaudio=yes \
+		brotli=no \
```

Comment:
Should this be `yes`?

Reply:
Yes, but then there is a problem - https://github.com/termux/termux-packages/actions/runs/5499118735